logger/logger.go

```diff
@@ -117,10 +122,37 @@ func (logger *Logger) log(verbosity Verbosity, message string) {
 // Print validation error.
 func (logger *Logger) PrintValidationErrors() {
 	if !logger.Quiet {
-		for _, err := range logger.validationErrors {
-			fmt.Println(err)
+		for _, resourceValidation := range logger.resourceValidation {
+			if len(resourceValidation.Errors) != 0 {
+				fmt.Println(resourceValidation.ResourceName)
+				for _, err := range resourceValidation.Errors {
+					fmt.Println("        ", err)
+				}
+			} else {
+				fmt.Println(resourceValidation.ResourceName, " has no validation errors")
+			}
+		}
+	}
+}
+
+// HasValidationErrors : Check if logger contains error messages
```

Comment:
I believe we do not need function name and a colon here for documentation purposes.